pkg/block/fetcher.go

```diff
@@ -231,14 +234,26 @@ func NewConcurrentLister(logger log.Logger, bkt objstore.InstrumentedBucketReade
 	}
 }
 
+func (f *ConcurrentLister) IsRetryableError(err error) bool {
```

Comment:
The location of this feels wrong since this is a minio related piece of code. What would be a proper place to put this?